mantidimaging/core/data/images.py

```diff
@@ -66,6 +66,20 @@ def metadata_save(self, f):
     def metadata_saves(self):
         return json.dumps(self.properties)
 
+    def record_parameters_in_metadata(self, func, *args, **kwargs):
+        if 'operation_history' not in self.properties:
+            self.properties['operation_history'] = []
+
+        def accepted_type(o):
+            return type(o) in [str, int, float, bool, tuple]
+
+        self.properties['operation_history'].append({
+            'name': func,
+            'args': [a if accepted_type(a) else None for a in args],
```

Comment:
Is it going to be a problem that this results in losing information?

Reply:
I see. Is the idea really just to have a list of operations that can be viewed.

Reply:
For now it is just to get the regions used when cropping the image so that the rotation centre parameter can be auto filled in the reconstruction dialog (the parameter must be relative to the cropped image, but the result given by the COR/Tilt dialog is relative to the entire image).

Reply:
:+1: